docs/designers-developers/developers/README.md

```diff
@@ -73,10 +35,10 @@ There are some advanced block features which require opt-in support in the theme
 
 ## Autocomplete
 
-Autocompleters within blocks may be extended and overridden. See [autocomplete](../docs/extensibility/autocomplete.md).
+Autocompleters within blocks may be extended and overridden. Learn more about the [autocomplete](../docs/designers-developers/developers/reference/hooks/autocomplete-filters.md) filters.
 
 ## Block Parsing and Serialization
 
 Posts in the editor move through a couple of different stages between being stored in `post_content` and appearing in the editor. Since the blocks themselves are data structures that live in memory it takes a parsing and serialization step to transform out from and into the stored format in the database.
 
-Customizing the parser is an advanced topic that you can learn more about in the [Extending the Parser](../docs/extensibility/parser.md) section.
+Customizing the parser is an advanced topic that you can learn more about in the [Extending the Parser](../docs/designers-developers/developers/reference/hooks/parser-filters.md) section.
```

Comment:
This and sibling links are broken:

- `reference/hooks` should become `filters`
- the relative path needs more depth (`../../docs`?)

Reply:
Good catch!